### DISCUSSION TITLE
How to increase repository sync history log retention period in ArgoCD using value.yaml

### DISCUSSION THREAD

**Discussion by vtandel-hub** [argoproj/argo-cd#13048] [General]
We have installed Argo CD in our AKS via public helm chart.

We can see the sync log is retaining only for 7 days. We need to increase it.

Can you please help us, how to do that?

Thanks

**Comment by jgwest** [comment]
Hi @vtandel-hub, my guess is you are hitting the `revisionHistoryLimit` limit on the Application `.spec`, which can be increased. 

Though if you do increase this value, note the warning that goes along with tha tfield:
```go
	// RevisionHistoryLimit limits the number of items kept in the application's revision history, which is used for informational purposes as well as for rollbacks to previous versions.
	// This should only be changed in exceptional circumstances.
	// Setting to zero will store no history. This will reduce storage used.
	// Increasing will increase the space used to store the history, so we do not recommend increasing it.
```

Beyond that, I'm not aware of any tunable parameters that would control sync history limit.
